### DISCUSSION TITLE
Error during ControlNet in i2i

### DISCUSSION THREAD

**Discussion by uzuuzu20002000** [vladmandic/sdnext#2583] [Q&A] [answered]
Thank you vladmandic for all your help. I am impressed with the beautiful images you are able to generate.

I have built the environment in paperspace and am trying to generate it with original.

I get an error during i2i's ControlNet

Is there some setting that is wrong?
I would be happy to receive your advice.

  Running script process:                                
                         extensions-builtin/sd-webui-controlnet/scripts/controln
                         et.py: IndexError    

※If I set the sampler to DEIS, etc., I can generate normally
╭───────────────────── Traceback (most recent call last) ──────────────────────╮
│ /notebooks/automatic/modules/scripts.py:519 in process                       │
│                                                                              │
│   518 │   │   │   │   args = p.per_script_args.get(script.title(), p.script_ │
│ ❱ 519 │   │   │   │   script.process(p, *args, **kwargs)                     │
│   520 │   │   │   except Exception as e:                                     │
│                                                                              │
│ /notebooks/automatic/extensions-builtin/sd-webui-controlnet/scripts/controln │
│ et.py:993 in process                                                         │
│                                                                              │
│    992 │   │   if not Script.process_has_sdxl_refiner(p):                    │
│ ❱  993 │   │   │   self.controlnet_hack(p)                                   │
│    994 │   │   return                                                        │
│                                                                              │
│ /notebooks/automatic/extensions-builtin/sd-webui-controlnet/scripts/controln │
│ et.py:982 in controlnet_hack                                                 │
│                                                                              │
│    981 │   │   t = time.time()                                               │
│ ❱  982 │   │   self.controlnet_main_entry(p)                                 │
│    983 │   │   if len(self.enabled_units) > 0:                               │
│                                                                              │
│ /notebooks/automatic/extensions-builtin/sd-webui-controlnet/scripts/controln │
│ et.py:747 in controlnet_main_entry                                           │
│                                                                              │
│    746 │   │   │   │                                                         │
│ ❱  747 │   │   │   │   input_image = [np.asarray(x)[:, :, 0] for x in input_ │
│    748 │   │   │   │   input_image = np.stack(input_image, axis=2)           │
│                                                                              │
│ /notebooks/automatic/extensions-builtin/sd-webui-controlnet/scripts/controln │
│ et.py:747 in <listcomp>                                                      │
│                                                                              │
│    746 │   │   │   │                                                         │
│ ❱  747 │   │   │   │   input_image = [np.asarray(x)[:, :, 0] for x in input_ │
│    748 │   │   │   │   input_image = np.stack(input_image, axis=2)           │

**Comment by vladmandic** [comment]
i've heard ppl mention this problem before, but nobody every created an issue for it and documented actual reproducible workflow - if i try with basic settings, i cannot reproduce which means its something workflow specific and i cannot click around to test every single possibilty.
i suggest to create an issue and document is as best as possible.

**Reply by uzuuzu20002000** [reply]
Yes, sir. That is certainly true.

Select image, type in prompt and negative, go to inpainting.
Put in the picture, mask it, select DPM++ 2M for Sampling method, choose brownian noise and discard penultimate sigma for Sampler options.
For Sigma algorithm, select Resize and fill as karras.
Resize as 1024*1024, seed as random, CFG scale as 30, CLIP skip as 2, Full quality, and Mask mode as Inpaint masked, Mask mode is set to Inpaint masked, Masked content is set to original, and Inpaint area is set to Only masked.
In ControlNet v1.1.419, in Unit0, check Enable and PixelPerfect, select IP-Adapter, Preprocessor is ip-adapter_clip_sd15, Model is, ip-adapter-plus-face_sd15 [71693645], Control Weight is set to 0.5, Dynamic Thresholding (CFG Scale Fix) is used, and FreeU is used with version 2 SD1.4.

If this does not reproduce the problem, I am very sorry.

**Reply by vladmandic** [reply, marked as answer]
likely a duplicate of #2568

**Reply by uzuuzu20002000** [reply]
ありがとうございました。気が付かず申し訳ないです。
状況を確認して、必要があればCNの方に連絡します。

**Reply by vladmandic** [reply]
english only please

**Reply by uzuuzu20002000** [reply]
Thank you very much.
We will check the situation and contact CN if necessary.
